### DISCUSSION TITLE
无法设置头像

### DISCUSSION THREAD

**Discussion by shmilyabyy** [halo-dev/halo#3881] [Q&A]
![image.png](https://vip2.loli.io/2023/04/29/OxsYTlkNzfZmMDt.png)
![image.png](https://vip2.loli.io/2023/04/29/LOp8MicgmNfa4un.png)
![image.png](https://vip2.loli.io/2023/04/29/CDLQR1wJ7EyKeFv.png)

**Comment by ruibaby** [comment]
已经在 https://github.com/halo-dev/halo/pull/3880 中修复，将在 2.5.2 发布。
